### ISSUE TITLE
SyntaxFest (and joint events generally)

### ISSUE THREAD

**Issue by nschneid** [acl-org/acl-anthology#2417]
It seems that joint.yaml described [on the wiki](https://github.com/acl-org/acl-anthology/wiki/Venues,-Volumes,-and-Events) no longer exists. So how are colocated events indicated?

In particular, the SyntaxFest events for 2019, 2021, 2023 under https://aclanthology.org/sigs/sigparse/ are handled inconsistently in terms of venue: two of the 2019 events display SyntaxFest as the venue, while the others display the workshop name (such as Depling). Ideally, both would be present.

**Comment by nschneid** [comment]
Aha, I found 0fac0a5 which describes a new approach.

The SyntaxFest 2019 paper data is under https://github.com/acl-org/acl-anthology/blob/master/data/xml/W19.xml. The four venues are W19-77, W19-78, W19-79, W19-80. I will try adding a second `<venue>` tag under each volume.

**Comment by xinru1414** [comment]
@mjpost this is also related to #2419 -- can a venue has multiple venue tags? For example `W19-78` is `tlt` and it also belongs to `SyntaxFest`. Which venue tag(s) should we use?

**Comment by akoehn** [comment]
Aha, I found 0fac0a5 which describes a new approach.
Should probably be updated in the docs — where in the wiki is the outdated info? — sent from my breakfast^tm

**Comment by mjpost** [comment]
Yes multiple venue tags are permitted. A volume will appear on the volumes page for any venue it has a venue tag for. The volumes pages used to list all real volumes as well as colocated workshops which was an incorrect use of the term. Now the volumes pages only list a venue's actual volumes; the events page lists the colocated ones. 

The conversion was tricky and heuristic-driven since joint used to indicate true joint events as well as colocated events. We should fix old venue associations as we come across them.

Note that a venue tag of "ws" means "this venue was assigned pre-2020 and we don't know its venue". there are too many to change all at once and many of them are one-off workshops that it's not work creating a venue for in retrospect.

**Comment by mjpost** [comment]
Agreed about updating the docs, @xinru1414 van you take a stab at it?

**Comment by nschneid** [comment]
bump - docs are still out of date

**Comment by mjpost** [comment]
Thanks for the reminder. I updated the docs.
